crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/Blobs.java

```diff
@@ -16,10 +16,13 @@
 
 package com.google.cloud.tools.crepecake.blob;
 
+import com.google.common.base.Charsets;
```

Comment:
StandardCharsets is preferred now.
Guava Charsets is a legacy of pre-Java 7.

Reply:
Filed #74 to change this in other parts as well.